### ISSUE TITLE
_index.md are not included?

### ISSUE THREAD

**Issue by timatlee** [Lednerb/bilberry-hugo-theme#263]
I have been trying to include some preamble for categories and tags, but finding that the theme doesn't seem to include the content from these files.

Referencing https://gohugo.io/templates/lists/#add-content-and-front-matter-to-list-pages

Using the `ananke` theme, my content directory can be structured as:
````
content
- page
--- _index.md
--- post1.md
--- post2.md
- categories/hugo
--- _index.md
````

The content of `page/_index.md` gets rendered at the top of the list of `post`s, and likewise the content of `categories/hugo/_index.md` gets rendered at the top of the list of the categories page.

When using the `bilberry-hugo-theme`, the contents of `_index.md` doesn't seem to be included at all. It's not in the raw HTML source either.

I'm not competent enough with hugo to figure out how to include the content. From the docs, it seems like it's a function of the hugo binary - so it should be included?

**Comment by igor-baiborodine** [comment]
@timatlee, that is correct. `_index.ntml` is ignored because of the current implementation of the [list template](https://github.com/Lednerb/bilberry-hugo-theme/blob/v2.8.0/layouts/_default/list.html).

I made a quick test with the Bilberry Sandbox. I modified the `list.html` template to include the content of the `_index.html` file and to list items the same way as they are listed in the archive page. I added an `_index.html` to the `content/audio/` folder to list all posts of the `Audio` type.

**list.html**: https://github.com/igor-baiborodine/bilberry-hugo-theme-sandbox/blob/issue/263-index-template-not-included/layouts/_default/list.html

**_index.html**: https://github.com/igor-baiborodine/bilberry-hugo-theme-sandbox/blob/issue/263-index-template-not-included/content/audio/_index.md

**test page**: https://deploy-preview-8--bilberry-sandbox.netlify.app/audio/

To change categories and tags lists, the [category.terms.html](https://github.com/Lednerb/bilberry-hugo-theme/blob/v2.8.0/layouts/_default/category.terms.html) and [tag.terms.html](https://github.com/Lednerb/bilberry-hugo-theme/blob/v2.8.0/layouts/_default/tag.terms.html) templates should be adjusted the same way.

I hope this helps. Let me know if you have any questions.
